traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -382,4 +373,12 @@ private static Error transformError(Error error) {
             return error;
         }
     }
+
+    private static String getSessionToken(Session session) {
+        Token token = session.getToken();
+        if (null == token) {
+            Log.w(TAG, "Session token is null");
```

Comment:
I'm using android.util.Log to be consistent with the rest of the class; but should we use TravelerLog.java throughout this class?